publish.sh

```diff
@@ -50,23 +51,34 @@ function build_centos9_base_image() {
   (cd cluster-provision/centos9 && ./build.sh)
 }
 
+function build_centos9_base_image_with_deps() {
+  IMAGE_TO_BUILD="$(find cluster-provision/k8s/* -maxdepth 0 -type d -printf '%f\n' | head -1)"
+  (cd cluster-provision/k8s/${IMAGE_TO_BUILD} && ../provision.sh)
+}
+
 function build_base_images() {
```

Comment:
I think we can inline this function and call `build_centos9_base_image_with_deps` directly, no?